### PR TITLE
Added Install hook and helper to run import on installation

### PR DESCRIPTION
**Pull request by martinyde**

### CODE REVIEW

File: os2forms_failed_jobs.install [hunk 1]
```diff
@@ -47,6 +49,13 @@ function os2forms_failed_jobs_schema(): array {
   return $schema;
 }
 
+/**
+ * Add existing queue elements to new table.
+ */
+function os2forms_failed_jobs_install(): void {
+  \Drupal::getContainer()->get(Helper::class)->handleImport();
```
Comment (rimi-itk) [inline os2forms_failed_jobs.install]: Use `Drupal::service(Helper::class)` to get the service as we do it throughout the OS2Forms project.

File: os2forms_failed_jobs.install [hunk 2]
```diff
@@ -90,3 +99,10 @@ function os2forms_failed_jobs_update_9001(): void {
 
   $schema->createTable($table_name, $table_schema);
 }
+
+/**
+ * Add existing queue elements to new table.
+ */
+function os2forms_failed_jobs_update_9002(): void {
+  \Drupal::getContainer()->get(Helper::class)->handleImport();
```
Comment (rimi-itk) [inline os2forms_failed_jobs.install]: Use `Drupal::service(Helper::class)` to get the service as we do it throughout the OS2Forms project.

**Review by rimi-itk — CHANGES_REQUESTED** [review]

File: os2forms_failed_jobs.install (hunk 1 above)
Comment (martinyde) [reply os2forms_failed_jobs.install]: Fixed

File: os2forms_failed_jobs.install (hunk 2 above)
Comment (martinyde) [reply os2forms_failed_jobs.install]: Fixed

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**